### PR TITLE
chore(IDX): move container to ci dir

### PR DESCRIPTION
**Pull request by cgundy**
This is the final PR to move all files from the dir `gitlab-ci` to `ci`. The container image was re-generated due to the new reference. No functional changes, just moving files and updating references.

### CODE REVIEW

**Review by basvandijk — APPROVED** [review]

File: rs/ethereum/cketh/mainnet/minter_proposal.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Verify that the hash of the gzipped WASM matches the proposed hash.
 ```
 git fetch
 git checkout 5ecbd59c6c9f9f874d4340f9fbbd96af07aa2576
-./gitlab-ci/container/build-ic.sh -c
```
Comment (gregorydemay) [inline rs/ethereum/cketh/mainnet/minter_proposal.md:35]: This should not change, this is a NNS proposal that was already executed.

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md [hunk 2]
```diff
@@ -38,6 +38,6 @@ Verify that the hash of the gzipped WASM matches the proposed hash.
 ```
 git fetch
 git checkout 667a6bd3bc08c58535b8b63bfebc01dba89c0704
-./gitlab-ci/container/build-ic.sh -c
```
Comment (gregorydemay) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md:41]: This should not change, this is a NNS proposal that was already executed.

File: rs/ethereum/cketh/playground/README.adoc [hunk 3]
```diff
@@ -97,7 +97,7 @@ If you deploy from a dev env,
 [source,shell]
 ----
 git checkout 5553ee5f7586290a59ed372a2b7aca847520af82
-./gitlab-ci/container/build-ic.sh -c
```
Comment (gregorydemay) [inline rs/ethereum/cketh/playground/README.adoc:100]: Since the line above references an exact commit, I would not change the instructions here.

**Review by nikolay-komarevskiy — APPROVED** [review]

**Review by gregorydemay — APPROVED** [review]
Thanks @cgundy !

**Review by andrewbattat — APPROVED** [review]

**Review by eichhorl — APPROVED** [review]

**Review by LittleChimera — APPROVED** [review]

**Review by kpop-dfinity — APPROVED** [review]

**Review by adambratschikaye — APPROVED** [review]

File: README.adoc [hunk 4]
```diff
@@ -100,11 +100,11 @@ Each https://dashboard.internetcomputer.org/releases[release proposal] includes
 
 To verify the build reproducibility of a specific `Elect Replica` proposal, you can just copy the one liner below to a fresh Ubuntu VM 22.04 or higher:
 
-    $ sudo apt-get install -y curl && curl --proto '=https' --tlsv1.2 -sSLO https://raw.githubusercontent.com/dfinity/ic/master/gitlab-ci/tools/repro-check.sh && chmod +x repro-check.sh && ./repro-check.sh -p <proposal>
+    $ sudo apt-get install -y curl && curl --proto '=https' --tlsv1.2 -sSLO https://raw.githubusercontent.com/dfinity/ic/master/ci/tools/repro-check.sh && chmod +x repro-check.sh && ./repro-check.sh -p <proposal>
```
Comment (sasa-tomic) [inline README.adoc:103]: Would you be okay if we merge this after the RC cut today? Otherwise we will have a problem with the NNS proposal summaries.

**Review by max-dfinity — APPROVED** [review]

**Pull request merged**